advanced/Scripts/piholeDebug.sh

```diff
@@ -946,6 +946,16 @@ process_status(){
     done
 }
 
+pihole-FTL_full_status(){
+    # if using systemd print the full status of pihole-FTL
+    echo_current_diagnostic "Pi-hole-FTL full status"
+    local FTL_status
+    if command -v systemctl &> /dev/null; then
+      FTL_status=$(systemctl status --full --no-pager pihole-FTL.service)
+      log_write "   ${FTL_status}"
+    fi
```

Comment:
Can you add in an `else` clause that states something like "systemctl not found"? Just something so that we know the `if` failed and it's not just an empty output from `systemctl`. 

```
Pi-hole-FTL full status

systemctl command not found.
```

instead of seeing

```
Pi-hole-FTL full status

<Output from next test>
```